### PR TITLE
Fixes being able to harmbaton people with an inactive baton

### PR DESCRIPTION
**Pull request by boy2mantwicethefam**
Problem is as such:
- A recent PR added a check in order to not allow blocked stunbatons on help intent stun
- The check was placed in the wrong spot, namely near the start of of the chain the actual stun is applied, meaning that the original sanity check (which was checking if the baton is active to stun) was doing almost nothing while the application of stun was now checking if the victim was a carbon
- With the way the code worked, being on help intent and hitting someone with an inactive baton would stop the whole thing from happening altogether, which was good. However, having the harm intent on would continue the proc and reach the carbon check.
- In short, if the victim was carbon (read: humans and what-else) and you had your harm intent on it would allow you to stun the victim with an inactive baton, bypassing frivolous mortal things such as whether the baton has enough battery power.

:cl:
 * bugfix: Stunbatons will no longer stun if the baton is off while harm intent is on.

### CODE REVIEW

**Comment by ShiftyRail** [comment]
Bullshit.

image

the unknown is not stunned.

**Comment by boy2mantwicethefam** [comment]
Stupid ass, you either forgot to update your repo to the latest commit or you tested the thing on my fix
stupid

**Comment by ShiftyRail** [comment]
this is from the live server.

**Comment by boy2mantwicethefam** [comment]
THE LIVE SERVER IS NOT UPDATED YET, check when the round started and when you merged the PR, the server only updates between rounds

**Comment by ShiftyRail** [comment]
Stand corrected on that, then.
